README.md

```diff
@@ -90,23 +91,24 @@ It's also important to remember that:
 
 #### Documentation about Previous Versions
-| SDK Version |          Alfresco Enterprise Version           |       Alfresco Community Version       | Documentation  |
-|-------------|:----------------------------------------------:|:--------------------------------------:|:-----|
-| SDK 4.5     |     Alfresco 7.0.x / 7.1.x / 7.2.x / 7.3.x     | Alfresco 7.0.x / 7.1.x / 7.2.x / 7.3.x | https://github.com/Alfresco/alfresco-sdk/tree/master/docs/README.md |
-| SDK 4.4     |         Alfresco 7.0.x / 7.1.x / 7.2.x         |     Alfresco 7.0.x / 7.1.x / 7.2.x     | https://github.com/Alfresco/alfresco-sdk/tree/master/docs/README.md |
-| SDK 4.3     | Alfresco 6.0.x / 6.1.x / 6.2.x / 7.0.x / 7.1.x |         Alfresco 7.0.x / 7.1.x         | https://github.com/Alfresco/alfresco-sdk/tree/master/docs/README.md |
-| SDK 4.2     |     Alfresco 6.0.x / 6.1.x / 6.2.x / 7.0.x     |             Alfresco 7.0.x             | https://github.com/Alfresco/alfresco-sdk/tree/master/docs/README.md |
-| SDK 4.1     |         Alfresco 6.0.x / 6.1.x / 6.2.x         |     Alfresco 6.0.x / 6.1.x / 6.2.x     | https://github.com/Alfresco/alfresco-sdk/blob/sdk-4.1/docs/README.md |
-| SDK 4.0     |             Alfresco 6.0.x / 6.1.x             |         Alfresco 6.0.x / 6.1.x         | https://github.com/Alfresco/alfresco-sdk/blob/sdk-4.0/docs/README.md |
-| SDK 3.1     |                 Alfresco 5.2.x                 |             Alfresco 5.2.x             | http://docs.alfresco.com/5.2/concepts/sdk-intro.html |
-| SDK 3.0     |                 Alfresco 5.2.x                 |             Alfresco 5.2.x             | http://docs.alfresco.com/5.2/concepts/sdk-intro.html |
-| SDK 2.2     |                 Alfresco 5.1.x                 |             Alfresco 5.1.x             | https://docs.alfresco.com/5.1/concepts/alfresco-sdk-intro.html |
-| SDK 2.1     |                 Alfresco 5.0.1                 |             Alfresco 5.0.d             | https://docs.alfresco.com/sdk2.1/concepts/alfresco-sdk-intro.html |
-| SDK 2.0     |                 Alfresco 5.0.0                 |             Alfresco 5.0.c             | https://docs.alfresco.com/sdk2.0/concepts/alfresco-sdk-intro.html |
-| SDK 1.1.1   |                 Alfresco 4.2.x                 |             Alfresco 4.2.x             | https://docs.alfresco.com/4.2/concepts/dev-extensions-maven-sdk-intro.html |
+| SDK Version |          Alfresco Enterprise Version           |           Alfresco Community Version           | Documentation                                                              |
+|-------------|:----------------------------------------------:|:----------------------------------------------:|:---------------------------------------------------------------------------|
+| SDK 4.5     | Alfresco 7.0.x / 7.1.x / 7.2.x / 7.3.x / 7.4.x | Alfresco 7.0.x / 7.1.x / 7.2.x / 7.3.x / 7.4.x | https://github.com/Alfresco/alfresco-sdk/tree/master/docs/README.md        |
```

Comment:
Should it be SDK 4.6?

Reply:
Of course it should be 4.6. Thank you for spotting it!